src/api.js

```diff
@@ -19,6 +19,20 @@ export const MOCK_DUOTONE_URLS = {
 export const MOCK_GROUP = {
 	id: 1234,
 	urlname: 'fake-hq-faff',
+	organizer: {
```

Comment:
Not sure if we'll need an organizer mock in isolation very much, so just manually populating it in `MOCK_GROUP` seems fine.

> Do we always prefer place kittens for photos?

HEY EVERYBODY THIS GUY HATES KITTENS!!!